code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm

```diff
@@ -15,23 +15,8 @@
 	layer = LARGE_MOB_LAYER //Looks weird with them slipping under mineral walls and cameras and shit otherwise
 
 /mob/living/simple_animal/hostile/megafauna/death(gibbed)
-	if(health > 0)
-		return
-	else
-		feedback_set_details("megafauna_kills","[initial(name)]")
-		..()
-
-/mob/living/simple_animal/hostile/megafauna/gib()
```

Comment:
Currently xenobio (when it is done) almost always heads towards at least one out of three end-game goals:
- Adamantine golem army
- Rainbowpotioning oneself into a headslug, becoming a changeling
- Getting a sentient magi/chaoscarp and going on a change rampage.
Given that all three require scientists to GET gold slimes, if not actually USE them (unless you get the rainbow to skip gold and go directly do adamantine), carps are a common by-product of getting headslugs. Having the 2500 health boss downed by a carp would be hilarious exactly once, before every scientist would start doing it.
Also, from my experience sec allows miners to do pretty much anything short of getting weapon crates, provided it is off-station, so SM-boss killing would become bad (especially since drake actually flies at you)